processing/src/main/java/io/druid/segment/StringDimensionIndexer.java

```diff
@@ -58,13 +58,18 @@
 
 public class StringDimensionIndexer implements DimensionIndexer<Integer, int[], String>
 {
-  private static final Function<Object, String> STRING_TRANSFORMER = o -> o != null ? o.toString() : null;
+  private static final Function<Object, String> EMPTY_TO_NULL_IF_NEEDED = o -> o != null
```

Comment:
changed to a method here. Since this is only used here, don;t see any value addition in moving it out to NulHandling, would like to keep it here only.